yarn.lock

```diff
@@ -9533,15 +9533,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"ast-types@npm:^0.13.2":
-  version: 0.13.4
-  resolution: "ast-types@npm:0.13.4"
-  dependencies:
-    tslib: ^2.0.1
-  checksum: 5a51f7b70588ecced3601845a0e203279ca2f5fdc184416a0a1640c93ec0a267241d6090a328e78eebb8de81f8754754e0a4f1558ba2a3d638f8ccbd0b1f0eff
-  languageName: node
-  linkType: hard
-
```

Comment:
When I search `ast-types` I get 3 results in `yarn.lock`.
`babel-plugin-react-docgen@npm`
`degenerator@npm`
`react-docgen@npm:^5.0.0`

I believe the package storybook uses is `react-docgen` should I limit this resolution to this package @Gudahtt?

![Screenshot 2023-02-14 at 10 38 05 AM](https://user-images.githubusercontent.com/8112138/218827759-977b8e99-cbc0-4ded-8098-3834546e8695.png)
![Screenshot 2023-02-14 at 10 38 10 AM](https://user-images.githubusercontent.com/8112138/218827761-6b60bffe-0c6a-4c50-926b-eae0fc30ca25.png)
![Screenshot 2023-02-14 at 10 38 16 AM](https://user-images.githubusercontent.com/8112138/218827762-b86ef654-bffb-476c-97c3-05a039257d02.png)

Reply:
I'm going to save you some time in the future:

```yarn info -A -R --dependents <package_name>```
![image](https://user-images.githubusercontent.com/4448075/218828655-3e973417-635d-438a-96a0-9ce8ec096b31.png)

Reply:
I think in this scenario its okay to resole them all to the higher version.

Reply:
I think I'm going to limit this to `@storybook/**/ast-types` so we know it's for storybook. Same as the original fix.

Reply:
Hmm  adding `"@storybook/**/ast-types"` to resolutions doesn't seem to update the yarn.lock and doesn't resolve the version of ast-types that storybook uses so I'm going to leave it unless otherwise instructed